### ISSUE TITLE
COM ACT and Standby not moving when using a device 

### ISSUE THREAD

**Issue by CerfontaineMathieu** [flybywiresim/aircraft#4030]
**Mod Version**
v0.5.4

**Describe the bug**

When using the mouse to change the standby COM, the com display is moving accordingly. However, If I use the keyboard or my Thrusmaster warthog, the COM display is stuck on 124.850.
Same behavior when switching com standby to com active 

**To Reproduce**

1. Only need a USB device and the MOD

**Expected behavior**

I should be able to see the com display incrementing accordingly to the buttons on my device.

**Actual behavior**

Display fixed

**References**

N/A

**Additional context**

N/A 
**Was this working before/when did the issue start occurring?**
Was working in v0.5.3
Is this a problem in the vanilla unmodded game?
No. Tried by uninstall the mod. It's working with the stock A320.

Discord username (if different from GitHub): Knoxville8596#8811

**Comment by Benjozork** [comment]
Have you tried the development version ?

**Comment by CerfontaineMathieu** [comment]
Yes the dev version is working, but I assume that is not supposed to be used except for troubleshoot and dev test ?

**Comment by Benjozork** [comment]
La version de développement est généralement utilisable pour les utilisateurs réguliers. Il risque d'avoir des problématiques par-ci par-là mais en moyenne ça fonctionne bien.

Le problème ici c'est que le correctif pour les panels radio externes est un changement majeur que l'on ne peut pas remettre sur une v0.5 à cause de sa complexité.

Nous allons bientôt préparer une v0.6 qui contient tous les changements de la dev incluant celui-ci.

**Comment by CerfontaineMathieu** [comment]
Ok Merci,
Je pensais la version de dev moins stable. 
Je vais rester dessus en attendant.

Merci

**Comment by MisterChocker** [comment]
Whats the situation here?

**Comment by CerfontaineMathieu** [comment]
For the now I’m using the dev branch. But I still have the issue on the v0.5.4
